corecomponent/androidcomponent/src/main/res/values/type.xml

```diff
@@ -49,4 +49,11 @@
         <item name="fontFamily">@font/notosans_regular</item>
         <item name="android:textColor">?colorOnBackground</item>
     </style>
+
+    <!-- Notosans Regular 16sp -->
+    <style name="TextAppearance.DroidKaigi.SessionMessage" parent="TextAppearance.MaterialComponents.Body1">
```

Comment:
Sorry. We don't use style for specific text 🙏 
But I will create an issue for that!

Reply:
Do you have created the issue ?
I will rewrite it ?
TextAppearance.DroidKaigi.SessionMessage
-> TextAppearance.DroidKaigi.Notice

Reply:
@takahirom 
Shall I rewrite ?
TextAppearance.DroidKaigi.SessionMessage
-> TextAppearance.DroidKaigi.Notice

Reply:
@ochim Oh.. I forgot it 🙏 Can you? please do it 🙏

Reply:
OK!  I try